File: TEST data for large repo/repo_Copy-2/Blueprism/BluePrism.Server.Domain.Models/ScheduleParameters.vb

Imports System.Runtime.Serialization
Imports BluePrism.Server.Domain.Models.DataFilters
Imports BluePrism.Server.Domain.Models.Pagination
Imports Func

<Serializable()>
<DataContract([Namespace]:="bp")>
Public Class ScheduleParameters
    <IgnoreDataMember>
    Public Name As DataFilter(Of String)
    <IgnoreDataMember>
    Public RetirementStatus As DataFilter(Of RetirementStatus)
    <IgnoreDataMember>
    Public ItemsPerPage As Integer
    <IgnoreDataMember>
    Public PagingToken As [Option](Of SchedulePagingToken)
End Class

File: TEST data for large repo/Repo_copy/Blueprism/BluePrism.Api/BluePrism.Api.Models/ScheduleParameters.cs
namespace BluePrism.Api.Models
{
    public class ScheduleParameters : IPagingModel<string>
    {
        public StartsWithStringFilterModel Name { get; set; }

        public CommaDelimitedCollection<RetirementStatus> RetirementStatus { get; set; }

        public int? ItemsPerPage { get; set; }

        public PagingTokenModel<string> PagingToken { get;set; }
    }
}
